sploosh-wasm/src/lib.rs

```diff
@@ -323,7 +314,7 @@ pub fn calculate_probabilities_with_board_constraints(
     let mut values = probabilities.iter().copied().collect::<Vec<_>>();
 
     // We sneak in the total probability at the end.
-    values.push(total_probability);
```

Comment:
how was this working before if it was off by a factor of 10^20?

Reply:
Oh, the need for this comes from my changes. Previously, the priors array was full of 1.0, so the total_probability ranged from 0.0 to about 1.0 depending on which boards were compatible. Now, since the array is full of 0.0, the total_probability ranges from 0.0 to about 1e-20. This is only returned for use in the opening optimizer, so I could fix it there, but this seemed simpler at the time.

Reply:
I added a comment to explain this multiplication in the source code.